hieradata/common/modules/designate.yaml

```diff
@@ -13,7 +13,8 @@ designate::use_syslog: true
 designate::default_transport_url:
   "rabbit://ceilometer:%{hiera('ceilometer_rabbit_password')}@%{hiera('service__address__rabbitmq_01')}:5672,ceilometer:%{hiera('ceilometer_rabbit_password')}@%{hiera('service__address__rabbitmq_02')}:5672,ceilometer:%{hiera('ceilometer_rabbit_password')}@%{hiera('service__address__rabbitmq_03')}:5672/ceilometer"
 designate::notification_transport_url:
-  "rabbit://designate:%{hiera('designate::rabbit_password')}@%{hiera('service__address__rabbitmq_01')}:5672,designate:%{hiera('designate::rabbit_password')}@%{hiera('service__address__rabbitmq_02')}:5672,designate:%{hiera('designate::rabbit_password')}@%{hiera('service__address__rabbitmq_03')}:5672/designate"
+  "rabbit://ceilometer:%{hiera('ceilometer_rabbit_password')}@%{hiera('service__address__rabbitmq_01')}:5672,ceilometer:%{hiera('ceilometer_rabbit_password')}@%{hiera('service__address__rabbitmq_02')}:5672,ceilometer:%{hiera('ceilometer_rabbit_password')}@%{hiera('service__address__rabbitmq_03')}:5672/ceilometer"
+#  "rabbit://designate:%{hiera('designate::rabbit_password')}@%{hiera('service__address__rabbitmq_01')}:5672,designate:%{hiera('designate::rabbit_password')}@%{hiera('service__address__rabbitmq_02')}:5672,designate:%{hiera('designate::rabbit_password')}@%{hiera('service__address__rabbitmq_03')}:5672/designate"
```

Comment:
Bør bruke service__transport__url her så blir det letter å oppdatere. ceilometer vhosten kommer sikkert til å bli flyttet til egen mq-server.

Reply:
Takk for tips, skal fikse.